BrainPortal/app/models/tool_config.rb

```diff
@@ -56,14 +56,19 @@ class ToolConfig < ActiveRecord::Base
                                    :message => "must be unique per pair [tool, server]" },
                   :if         => :applies_to_bourreau_and_tool?
 
+  validates                 :container_index_location,
+                            :format => { :with => /^[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.[a-z]{2,6}$/i },
```

Comment:
Failed to validate "shub://" as an index. Should allow it, I had this format of regex: 

`:format => { :with => /^[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.[a-z]{2,6}$|^shub\:\/\/$/}`